packages/blade/src/components/Switch/switchTokens.ts

```diff
@@ -106,15 +91,15 @@ const switchMotion: Record<string, Record<string, `motion.${EasingString | Durat
     thumbIcon: 'motion.duration.xquick',
     track: 'motion.duration.xquick',
   },
-};
+} as const;
 
-const switchHoverTokens: SelectorInputHoverTokens = {
+const switchHoverTokens = {
   default: {
     background: {
       checked: 'colors.brand.primary.600',
       unchecked: 'colors.brand.gray.600.lowContrast',
```

Comment:
we don't have these tokens anymore no 🤔

Reply:
Oops, good catch. Forgot to make the hover tokens typesafe in SelectorInput and missed this.